DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -226,23 +226,28 @@ function InstallPackage
 
     $env:Path = [Environment]::GetEnvironmentVariable('Path','Machine')
 
-    [string]$chocoinstallparams = '-y'
+    [string]$chocoParams = '-y'
     if ($pParams) {
-        $chocoinstallparams += " --params=`"$pParams`""
+        $chocoParams += " --params=`"$pParams`""
     }
     if ($pVersion) {
-        $chocoinstallparams += " --version=`"$pVersion`""
+        $chocoParams += " --version=`"$pVersion`""
     }
     if ($pSource) {
-        $chocoinstallparams += " --source=`"$pSource`""
+        $chocoParams += " --source=`"$pSource`""
     }
     if ($cParams) {
-        $chocoinstallparams += " $cParams"
+        $chocoParams += " $cParams"
+    }
+    # Check if Chocolatey version is Greater than 0.10.4, and add --no-progress 
+    if ((Get-ChocoVersion) -ge [version]('0.10.4')){

```

Comment:
Not to be too difficult, but fully qualify this. I've run into issues where things were not fully qualified causing issues in different versions of PowerShell. `[System.Version]` 

Thanks!

Reply:
will do